### ISSUE TITLE
SolrConnection list method doesn't work as expected with solr 5.4

### ISSUE THREAD

**Issue by azenk** [solrcloudpy/solrcloudpy#25]
It looks like there may be a bug with solr 5.4.  See below:

```
$ cat solrtest.py 
def main():
    conn = solrcloudpy.connection.SolrConnection(["localhost:8983"])
    print(conn.list())
```

When I run the test script against 5.4 (docker solr:latest + zookeeper) I get the following:
```
$ python solrtest.py 
[u'/aliases.json', u'/clusterstate.json', u'/collections', u'/live_nodes', u'/overseer', u'/overseer_elect', u'/security.json', u'/zookeeper']
```

When I run the test script against 5.3 (docker solr:5.3 + zookeeper) I get the following (as expected):
```
$ python solrtest.py 
[u'collection1']
```

**Comment by dfdeshom** [comment]
Not sure if this is a bug or if the API has changed in solr 5.4. If the solr API has changed, please submit a  patch to `solrcloudpy` so it gives the right answer.

**Comment by azenk** [comment]
Yeah, I assume it's an api change.  I'm not really a solr user, so I'm not sure when I'd get around to writing a patch.  Just thought somebody should be aware of the problem.

**Comment by relwell** [comment]
Hi @azenk, I've introduced a changeset in my personal fork to fix this problem: https://github.com/relwell/solrcloudpy/commit/ba69a3ae55419905eea04bddb5dc93fddd4bc725

**Comment by relwell** [comment]
Fixed by #26
